src/cross_module_fn.h

```diff
@@ -129,6 +129,8 @@ typedef struct CrossModuleFunctions
 	Datum (*chunk_set_default_data_node)(PG_FUNCTION_ARGS);
 	Datum (*create_chunk)(PG_FUNCTION_ARGS);
 	Datum (*show_chunk)(PG_FUNCTION_ARGS);
+	Datum (*drop_chunk_replica)(PG_FUNCTION_ARGS);
+
```

Comment:
Optional: would recommend using `PGFunction `, but we aren't consistent about this in existing code, so we could just clean up later.

Note the extra line, however.

```suggestion
	Datum (*drop_chunk_replica)(PG_FUNCTION_ARGS);
```

Reply:
Done. Using `PGFunction` now.